index.js

```diff
@@ -799,3 +791,30 @@ module.exports.fromElement = fromElement;
 module.exports.toElement = toElement;
 module.exports.split = split;
 module.exports.concat = concat;
+
+/**
+ * merge custom conversion table
+ * note: to remove a default conversion, simply pass <TAGNAME>: null
+ * note: calling this with no obj sets the default conversions
+ * @param {object} [obj] tagname mappings
+ */
+module.exports.setSameAs = function (obj) {
+  if (obj) {
+    sameAs = _.assign(sameAs, obj);
+  } else {
+    sameAs = {
+      B: 'STRONG',
+      U: 'EM',
+      I: 'EM',
+      H1: 'H2',
+      H3: 'H2',
+      H4: 'H2',
+      H5: 'H2',
+      H6: 'H2',
+      STRIKE: 'DEL'
+    };
+  }
+};
+
+// when this library is first loaded, set the default conversions
+module.exports.setSameAs();
```

Comment:
Might be simpler to define the default value in the initial declaration of `sameAs`, and then make the `obj` param required for the `setSameAs` method.